### PR TITLE
Added testHelper functionality in e2e tests

### PR DESCRIPTION
**Pull request by rajkumarpb**
Changes needed to align with PR #55 in https://github.com/dylants/price-finder

### CODE REVIEW

File: generators/app/templates/site-uris-test.js [hunk 1]
```diff
@@ -38,11 +27,7 @@ describe('price-finder for <%= upperCaseSite %> Store URIs', () => {
     it('should respond with a price, and the right category and name for findItemDetails()', (done) => {
       priceFinder.findItemDetails(uri, (err, itemDetails) => {
         expect(err).toBeNull();
-        expect(itemDetails).toBeDefined();
-
-        verifyPrice(itemDetails.price);
-        verifyName(itemDetails.name, '');
-        verifyCategory(itemDetails.category, '');
+        verifyItemDetails(itemDetails, 'PixelJunk™ Monsters', 'Video Games');
```
Comment (dylants) [inline generators/app/templates/site-uris-test.js]: Please remove 'PixelJunk™ Monsters' and 'Video Games' and instead leave them as empty strings ('').

File: generators/app/templates/site-uris-test.js [hunk 2]
```diff
@@ -3,24 +3,13 @@
...
-function verifyName(actualName, expectedName) {
-  expect(actualName).toEqual(expectedName);
-}
-
-function verifyCategory(actualCategory, expectedCategory) {
-  expect(actualCategory).toEqual(expectedCategory);
-}
+const priceFinder = testHelper.priceFinder;
+const verifyPrice = testHelper.verifyPrice;
+const verifyItemDetails = testHelper.verifyItemDetails;
```
Comment (dylants) [inline generators/app/templates/site-uris-test.js]: It looks like the consts are there multiple times?

File: generators/app/templates/site-uris-test.js [hunk 3]
```diff
@@ -3,24 +3,13 @@
 // set the timeout of these tests to 10 seconds
 jasmine.getEnv().defaultTimeoutInterval = 10000;
```
Comment (dylants) [inline generators/app/templates/site-uris-test.js]: The timeout should not be here since it's in the test helper

**Comment by dylants** [comment]
We might want to just hold off on this PR until the other one in price-finder is merged.  And this is missing the new test helper file that was added.  But I appreciate the thought!

**Comment by rajkumarpb** [comment]
Should we have the testHelper class in here? I think that should be there in price-finder repo and not here! Correct me if I am wrong.

**Comment by dylants** [comment]
@rajkumarpb Yes, you're right!  I was not thinking clearly last night, not enough sleep!  This should continue to add the 3 files as before.

**Comment by dylants** [comment]
@rajkumarpb I've reopened the pull request if you'd like to make the changes I mentioned above.

**Comment by rajkumarpb** [comment]
@dylants Done with commits. Please merger the PR!

**Comment by dylants** [comment]
Looks great, thanks!

**Pull request merged**